Orange/data/util.py

```diff
@@ -18,11 +18,15 @@ def one_hot(values, dtype=float):
     result
         2d array with ones in respective indicator columns.
     """
+    if not len(values):
+       return np.zeros((0, 0), dtype=dtype)
     return np.eye(int(np.max(values) + 1), dtype=dtype)[np.asanyarray(values, dtype=int)]
 
 
 def scale(values, min=0, max=1):
     """Return values scaled to [min, max]"""
+    if not len(values):
+        return 1
```

Comment:
Fixed. Explaining what I mis-thought here would be a bit embarrassing, so I'm skipping this part.